concepticondata/conceptlists/Cross-1964-241.tsv

```diff
@@ -98,7 +98,7 @@ Cross-1964-241-96	96	heart	1223	HEART
 Cross-1964-241-97	97	heavy	1210	HEAVY
 Cross-1964-241-98	98	hide — conceal	602	HIDE (CONCEAL)
 Cross-1964-241-99	99	hill	614	HILL
-Cross-1964-241-100	100	hit — strike (with hand)	353	STRIKE
+Cross-1964-241-100	100	hit — strike (with hand)	3309	STRIKE (sOMETHING)
```

Comment:
Why would this one be STRIKE (SOMETHING)? Does it specify somewhere that the target can't be a person?